pysatNASA/instruments/maven_sep.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Module for the MAVEN sep instrument.
+
+Supports the Solar Energetic Particle (SEP) data from
+onboard the Mars Atmosphere and Volatile Evolution (MAVEN) satellite.
+
+Accesses local data in CDF format.
+Downloads from CDAWeb.
+
+Properties
+----------
+platform
+    'maven'
+name
+    'sep'
+tag
+    None supported
+inst_id
+    ['s1', 's2']
+
+Examples
+--------
+::
+    import pysat
+
+    insitu = pysat.Instrument(platform='maven', name='sep', inst_id='s1')
+    insitu.download(dt.datetime(2020, 1, 1), dt.datetime(2020, 1, 31))
+    insitu.load(2020, 1, use_header=True)
+
+"""
+
+import datetime as dt
+import functools
+
+from pysat.instruments.methods import general as mm_gen
+from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+from pysatNASA.instruments.methods import maven as mm_mvn
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'maven'
+name = 'sep'
+tags = {'': 'Level 2 Solar Energetic Particle data'}
+inst_ids = {'s1': [''], 's2': ['']}
```

Comment:
This would be better as 'l2' for the tag, as a future-proofing effort.

Reply:
I think most instruments have used l2 (level-2) public science data as a default. 
Does l2 mean something different here than say C/NOFS or ICON?

Reply:
It means extra typing for end-users, and there is no plan to currently bring other levels in.